### DISCUSSION TITLE
Aargaudütsch

### DISCUSSION THREAD

**Discussion by esthicodes** [Estheryu991/SwissGerman_Dictionary#31] [General]
Besser vom Grosse is Detail goh

Maschinells lerne

D Methode vom DiscountCalculator gebe je en zahl zrogg

luut denke prozesskommentiere

**Comment by esthicodes** [comment]
- us trotz schriib ich dir ned zrugg

- die medaille esch es vermächtnis vo mim grosspapi

- minn vorgsetzte hed en seehr unkonventionelli art


- sie hed en seehr individuelle stil

- vo dem punkt uus esch d rigi ned sichtbar

- schwesterlichkeit esch mir es wichtigs aaliege

**Comment by esthicodes** [comment]
Muess warte es hann kei Wlan.

muesch Warte, es het kei Wlan.

**Comment by esthicodes** [comment]
was machemer und was isch de plan? :)
-  gli

**Comment by esthicodes** [comment]
![image](https://user-images.githubusercontent.com/78131082/197239885-fc5a3132-d760-47da-ba76-4818a634c3d4.jpeg)

**Comment by esthicodes** [comment]
eine isch Uf ds bini äbe när o cho 😉👍🏻 debLeitig gstande! nüt me gooht
![Cargando 197285928...]
die Türen sind  nicht zugegegen und darum ist der Zug nicht weitergefahren

**Comment by D3strukt0r** [comment]
vs stories isch geil wil 

- sie mache min code als stories wie bi snapchat
- technisch bruchen wir javascript

**Comment by yveswehrli** [comment]
Nimmsch au no eh Wägbier?

**Comment by yveswehrli** [comment]
Ich programmiere gärn Javascript

**Comment by esthicodes** [comment]
Ich fahr jede Tag vom Aargau nach Züri

**Comment by esthicodes** [comment]
`lass ich das als erzeugend System`
`erzeugend ` : `erzügend`

**Comment by esthicodes** [comment]
lineare algebra: 
uf Argaudütsch: 
Menge ein unabhängigen Vektoren us einem endlich dimensional Vektoren lässt sich ergänzen zu einenr Basis vo V. 

Bewis: 
`{v_1 ... v_n} ein unabhängigkeit gsi. `

**Comment by esthicodes** [comment]
Korrelation VektorRaum: Sei V ein n dimensional VektorRaum Dann isch jede Menge von n lineare unabhängigen Vektoren eine Basis mache. 

Bewis isch ein Satz.

Zämäfassig: Jede Basis hat n Vektoren. 

1. Mehr als n Vektoren in V sind lineare abhängig. 
2. Weniger als n Vektoren in V sind nöd erzeugend (erzügend / ).
3. N Vektoren in V sind lineare abhängig. 
4. unabhängig <=> sie sind Erzeugend 
5. <=> sue bliebe eine Basis 

1. Mehr als n Vektoren in V sind lineare abhängig. 
2. Weniger als n Vektoren in V sind nöd erzeugend (erzügend / ).
3. N Vektoren in V sind lineare abhängig. 
4. unabhängig <=> sie sind Erzügend 
5. <=> sue bliebe en Basis
